ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -14,50 +15,36 @@
 
 
 @st.composite
-def _arrays_idx_n_dtypes(draw):
-    num_dims = draw(st.shared(helpers.ints(min_value=1, max_value=4), key="num_dims"))
-    num_arrays = draw(
-        st.shared(helpers.ints(min_value=2, max_value=4), key="num_arrays")
-    )
-    common_shape = draw(
-        helpers.lists(
-            arg=helpers.ints(min_value=2, max_value=3),
-            min_size=num_dims - 1,
-            max_size=num_dims - 1,
-        )
-    )
-    unique_idx = draw(helpers.ints(min_value=0, max_value=num_dims - 1))
-    unique_dims = draw(
-        helpers.lists(
-            arg=helpers.ints(min_value=2, max_value=3),
-            min_size=num_arrays,
-            max_size=num_arrays,
-        )
+def _concat_helper(draw):
+    dtype = draw(st.sampled_from(ivy_np.valid_dtypes))
+    shape = list(
```

Comment:
this is not a general shape, this is **always** `(1,)`, it looks like the current `_arrays_idx_n_dtypes` is more general. What's the rationale behind removing `_arrays_idx_n_dtypes`?

Reply:
Apologies, those were temporary values I was using while trying to diagnose issues with Container.concat. `_arrays_idx_n_dtypes` was replaced with a composite strategy I could more easily wrap my head around while doing so, and I forgot to replace it here. Reimplementing the original strategy now.